org.gnome.Papers.json

```diff
@@ -0,0 +1,181 @@
+{
+    "app-id": "org.gnome.Papers",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "default-branch": "stable",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "papers",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=pulseaudio",
```

Comment:
does a PDF reader really need sound access?

Reply:
I know the PDF spec is scary. You can put lots of things in there, including embedding videos and such. Most useful probably to play sound during presentations (evince also has it: https://github.com/flathub/org.gnome.Evince/blob/master/org.gnome.Evince.json#L12)

Reply:
Fair enough.